drizzle-kit/src/cli/commands/introspect.ts

```diff
@@ -125,20 +125,20 @@ export const introspectPostgres = async (
 
 	render(
 		`[${
-			chalk.green(
+			pico.green(
 				'✓',
 			)
-		}] You schema file is ready ➜ ${chalk.bold.underline.blue(schemaFile)} 🚀`,
+		}] You schema file is ready ➜ ${pico.blue(pico.bold(pico.underline(schemaFile)))} 🚀`,
```

Comment:
The `picocolors` don't supports the chained syntax:

```js
- `[${chalk.green(}] You schema file is ready ➜ ${chalk.bold.underline.blue(schemaFile)} 🚀`

+ `[${pico.green(}] You schema file is ready ➜ ${pico.blue(pico.bold(pico.underline(schemaFile)))} 🚀`
```

The nested calls `pico.blue(pico.bold(pico.underline(schemaFile)))` have worstly readable and much slower than `chalk.bold.underline.blue(schemaFile)`.

> [!TIP]
> The smaller and faster "drop-in" replacement for `chalk` is [ansis](https://github.com/webdiscus/ansis).

Ansis supports the [named import](https://github.com/webdiscus/ansis?tab=readme-ov-file#named-import), [chained syntax](https://github.com/webdiscus/ansis?tab=readme-ov-file#chained-syntax) and [nested template strings](https://github.com/es-tooling/ecosystem-cleanup/issues/132#nested-syntax).

So that you can write **shorter** and **more readable** code, for example:

```js
const { green, red, bold } = require('ansis');

- `[${pico.green(}] You schema file is ready ➜ ${pico.blue(pico.bold(pico.underline(schemaFile)))} 🚀`
+ `[${green(}] You schema file is ready ➜ ${bold.underline.blue(schemaFile)} 🚀`

// an example of nested template strings:
red`it contains chained ${bold.blue`sub`} strings`;
```

See [Compare features](https://github.com/drizzle-team/drizzle-orm/pull/2736/files#compare) of most popular libs.

## Package size

Ansis is one of the smallest packages, see please [Compare package sizes](https://github.com/es-tooling/ecosystem-cleanup/issues/132#compare-size) of most popular libs.

Unpacked Size: 
- [chalk](https://www.npmjs.com/package/chalk) (43.7 kB)
- [ansis](https://www.npmjs.com/package/ansis) (11.3 kB) - with the same functionality as chalk (plus support for CJS/ESM)
- [picocolors](https://www.npmjs.com/package/picocolors) (6.37 kB)

## Performance

See the [Benchmarks](https://github.com/drizzle-team/drizzle-orm/pull/2736/files#benchmark) of most popular libs.

> [!CAUTION]
>
> Using the nested calls like `pico.blue(pico.bold(pico.underline(schemaFile)))` is much slower than using chined syntax.
> Therefore using `picocolors` is 2x **slower** in such use cases.
>
> For example, simple test. We will apply 3 styles: `red`, `bold`, `underline`.
>
> Using `picocolors`:
> ```js
> pico.red(pico.bold(pico.underline('foo')))
> ```
>
> Using `chalk`:
> ```js
> chalk.red.bold.underline('foo')
> ```
> 
> Using `ansis`:
> ```js
> red.bold.underline('foo')
> ```
>
> Result of  "benchmark@2.1.4":
>
> ```diff
> + ansis       9,203,260 ops/sec ± 0.35%
> - chalk       6,115,064 ops/sec ± 0.22%
> - picocolors  4,467,594 ops/sec ± 0.16%
> ```